python/lsst/ip/diffim/dipoleAlgorithms.cc

```diff
@@ -158,6 +161,9 @@ void wrapDipoleAlgorithms(lsst::cpputils::python::WrapperCollection &wrappers) {
     declarePsfDipoleFlux(wrappers);
 }
 
+// Remove these pragmas on DM-44030
+#pragma GCC diagnostic pop
+
```

Comment:
Could you add a comment on what these `pragma` s are doing?

Reply:
They're from the C++ deprecation docs: https://developer.lsst.io/stack/deprecating-interfaces.html#c-deprecation